cmake/open62541ppConfig.cmake.in

```diff
@@ -0,0 +1,17 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/open62541ppTargets.cmake")
+
+set (open62541pp_TOOLS_DIR @PACKAGE_open62541pp_install_tools_dir@ CACHE PATH "Path to the directory that contains the tooling of the stack")
+set (open62541pp_NODESET_DIR @PACKAGE_open62541pp_install_nodeset_dir@ CACHE PATH "Path to the directory that contains the OPC UA schema repository")
```

Comment:
Do we need this? The open62541 tooling and OPC UA schema should be part of the open62541 package and not ours?

Reply:
Lines 5 and 6 were not created by me, maybe I forgot to remove them when I rebased against your master branch?

Reply:
No we didn't, this is a leftover from the copy I did of the original open62541 template file